packages/lesswrong/components/posts/PostsPage/T3AudioPlayer.tsx

```diff
@@ -0,0 +1,64 @@
+// TODO: Import component in components.ts
+import React, { useRef } from 'react';
+import { registerComponent } from '../../../lib/vulcan-lib';
+import { useTracking } from "../../../lib/analyticsEvents";
+import Helmet from 'react-helmet';
+import classNames from 'classnames';
+import { useEventListener } from '../../hooks/useEventListener';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  embeddedPlayer: {
+    marginBottom: "30px"
+  },
+  hideEmbeddedPlayer: {
+    display: "none"
+  },
```

Comment:
Deleted from `PostsPage`?

Reply:
These are actually still used on PostsPage (for the other audio embed)